<launch>

  	<node 	pkg="bebop2_state_estimator"
    		type="full_state_kalman_filter_node"
    		name="full_state_kalman_filter_node_1"
    		respawn="false"
    		launch-prefix=""
    		required="true">

			<remap from="/bebop/pose" to="/Bebop1/pose"/>
			<remap from="/bebop/full_state_estimation" to="/Bebop1/full_state_estimation"/>
  	</node>

</launch>
